File: 프로그래머스/0/120853. 컨트롤 제트/컨트롤 제트.py
def solution(s):
    answer = 0
    str_list = s.split(" ")
    arr = [0] * len(str_list)
    
    for i in range(len(arr)):
        if str_list[i] == "Z":
            arr[i] = int(str_list[i-1])
            answer -= arr[i]
        else:
            arr[i] = int(str_list[i])
            answer += arr[i]
            
    return answer

File: 프로그래머스/0/120853. 컨트롤 제트/컨트롤 제트.js
function solution(s) {
    let answer = 0;
    const strList = s.split(" ");
    const arr = new Array(strList.length).fill(0);
    
    for (let i = 0; i < arr.length; i++) {
        if (strList[i] === "Z") {
            arr[i] = parseInt(strList[i - 1]);
            answer -= arr[i];
        } else {
            arr[i] = parseInt(strList[i]);
            answer += arr[i];
        }
    }
    
    return answer;
}
